components/table/SubTableRow.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { CompatibilitiesType } from '../../service/types';
+import BBImage from './BuildingBlocksImage';
+
+type Props = {
+  compatibilities: CompatibilitiesType[];
+};
+
+const SubTableRow = ({ compatibilities }: Props) => {
```

Comment:
We could rename it to `SubTableRows` as multiple rows are created. Additionally, maybe it'd be better Idea to have separate components for SubTableRows (using list of compatibilities to create SubTableRow JSXs) and SubTableRow (using CompatibilityType json to create single row)

Reply:
This is component for one row.